### PR TITLE
docs(data): add ng add installation

### LINKED ISSUE #1554: Build: add package for ngrx/data
Create initial package and setup to produce nightly builds of `@ngrx/data` from John Papa and Ward Bell as an official package.

Source: https://github.com/johnpapa/angular-ngrx-data

Checklist

- [ ] Add build setup to modules folder
- [x] Setup github repo for night builds
- [ ] Create doc for installation and usage

### PR DESCRIPTION
**Pull request by timdeschryver**
## PR Checklist

Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/ngrx/platform/blob/master/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [x] Documentation has been added / updated (for bug fixes / features)

## PR Type

What kind of change does this PR introduce?

```
[ ] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Build related changes
[ ] CI related changes
[x] Documentation content changes
[ ] Other... Please describe:
```

## What is the current behavior?

I think this is the last step in order to close #1554

## Does this PR introduce a breaking change?

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: projects/ngrx.io/content/guide/data/install.md [hunk 1]
```diff
@@ -15,3 +15,29 @@ For more information on using `yarn` check out the docs <a href="https://yarnpkg
...
+
+This command will automate the following steps:
+
+1. Update `package.json` > `dependencies` with `@ngrx/data`.
+2. Run `npm install` to install those dependencies.
+3. Update your `src/app/app.module.ts` > `imports` array with `EntityDataModule` or `EntityDataModuleWithoutEffects` depending on the `effects` flag.
+
+With the `migrateNgRxData` flag the following will also take place:
+
+1. remove `angular-ngrx-data` from `package.json` > `dependencies`.
```
Comment (brandonroberts) [inline projects/ngrx.io/content/guide/data/install.md]:
```suggestion
1. Remove `angular-ngrx-data` from `package.json` > `dependencies`.
```

File: projects/ngrx.io/content/guide/data/install.md [hunk 2]
```diff
@@ -15,3 +15,29 @@ For more information on using `yarn` check out the docs <a href="https://yarnpkg
...
+2. rename `angular-ngrx-data` types to the matching `@ngrx/data` types.
```
Comment (brandonroberts) [inline projects/ngrx.io/content/guide/data/install.md]:
```suggestion
2. Rename `angular-ngrx-data` types to the matching `@ngrx/data` types.
```

**Comment by johnpapa** [comment]
this is awesome!

Where should I make doc PRs - as I see a few tweaks I would like to suggest?

**Comment by brandonroberts** [comment]
The docs files are here https://github.com/ngrx/platform/tree/master/projects/ngrx.io/content/guide/data

**Comment by johnpapa** [comment]
ok - one example `idiosyncratic` - I think we should use a more widely known word :)

I'll do a run through later.

also - i'd like to update my labs with this, once it's ready to go. What's the timing for making it go live?

**Comment by brandonroberts** [comment]
It's ready to test with if you want to try out the latest beta, currently `8.0.0-beta.1`. I don't foresee many changes to it other than docs

**Pull request merged**